content/en/agent/faq/circleci-incident-impact-on-datadog-agent.md

```diff
@@ -3,126 +3,122 @@ title: Impact of the CircleCI Security Incident on the Datadog Agent
 
-To delete the key from the RPM database and stop trusting it, run the following command:
+Ensure that your hosts **stop trusting the affected key**. If the previous steps indicated that a host trusts the key, follow these steps:
 
-```bash
-$ sudo rpm --erase gpg-pubkey-4172a230-55dd14f6
-```
+1. If you use configuration automation tools or plugins, such as the Ansible Datadog role, update them to the latest version listed in [What Datadog is doing](#what-datadog-is-doing).
 
-To delete the key from the Datadog repo file, remove the `gpgkey` line that ends with `DATADOG_RPM_KEY.public`. If this was the only `gpgkey` entry in your repo file, replace it with `https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public`. Note that this has implications explained in the section [Implications of no Longer Trusting the Affected Key](#implications-of-no-longer-trusting-the-affected-key).
+   Remaining on older versions of these automation tools or plugins might reverse your remediation efforts. If you can't yet update to the new fixed versions, add the manual key deletion steps (step 3 and 4) to your automation tool runbooks, and ensure these run _after_ the Datadog tools and plugins in your runbook order.
 
-#### Usage of automation tools
+2. For hosts that are set up using the official Datadog install scripts, run the latest version of the install script to untrust the key and provision the updated repo files.
 
-If you're using an automation tool or plugin, such as the Ansible Datadog role, update to the latest version (see the full list with updated versions in [What Datadog is doing to mitigate the implications](#what-datadog-is-doing-to-mitigate-the-implications))).
+3. If running `rpm -q gpg-pubkey-4172a230-55dd14f6` still detects the key, delete the key from the RPM database and stop trusting it by running the following command:
 
-Older versions of these automation tools or plugins might reverse the manual steps recommended above. If you can't yet update to the new versions that fix this, we recommend adding these manual steps to your automation tool runbooks.
+   ```bash
+   $ sudo rpm --erase gpg-pubkey-4172a230-55dd14f6
+   ```
 
-### Verify installed packages
+4. If `DATADOG_RPM_KEY.public` is still listed in the repo file, delete the key by removing the `gpgkey` line that ends with `DATADOG_RPM_KEY.public`. If this is the only `gpgkey` entry in your repo file, replace it with `https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public`. Read more in the section [Implications of no longer trusting the affected key](#implications-of-no-longer-trusting-the-affected-key).
 
-Out of an abundance of caution, Datadog also recommends that, on your impacted systems, you verify that all packages signed by the affected key were built by Datadog. You can run [this script][3] to verify that:
+5. Out of an abundance of caution, verify that Datadog built the packages on the affected hosts signed by the affected key by running [this script][3]:
```

Comment:
I think the following is "more correct":

```suggestion
5. Out of an abundance of caution, verify that Datadog built the packages signed by the affected key on the affected hosts by running [this script][3]:
```

i.e.: we want to verify that, on the affected hosts, Datadog built the packages signed by the affected key.